src/metatensor/models/cli/train.py

```diff
@@ -22,6 +22,7 @@
 from .. import CONFIG_PATH
 from ..utils.data import get_all_species, read_systems, read_targets
 from ..utils.data.dataset import _train_test_random_split
+from ..utils.device import get_available_devices, pick_devices
```

Comment:
maybe utils.devices? The functions are always plural.

Reply:
Sure